pyprep/find_noisy_channels.py

```diff
@@ -24,15 +24,24 @@ class NoisyChannels:
 
     """
 
-    def __init__(self, raw):
+    def __init__(self, raw, do_detrend=True):
         """Initialize the class."""
         # Make sure that we got an MNE object
         assert isinstance(raw, mne.io.BaseRaw)
 
         self.raw_mne = raw.copy()
         self.sample_rate = raw.info["sfreq"]
+        if do_detrend:
+<<<<<<< HEAD
+            self.raw_mne._data = removeTrend(
+                self.raw_mne.get_data(), sample_rate=self.sample_rate
+            )
+
+=======
+            self.raw_mne._data = removeTrend(self.raw_mne.get_data(), sample_rate=self.sample_rate)
+        
+>>>>>>> f249828a965fe3467462683228fa77092f21816b
```

Comment:
seems like you accidentally left some `git` merge stuff in here like ` >>>>>>> f249828a965fe3467462683228fa77092f21816b` and also on lines 35 and 40

Reply:
sorry indeed Im not the most versed using git :) , I will try to solve this stuff now to get this out.

Reply:
> sorry indeed Im not the most versed using git :)

you're in the best place to get better :-) if you run into problems, you can ask me anything

Reply:
Solved this, apparently i just didnt save the resolved file in vscode so I pushed the merge conflict  stuff